### PR TITLE
Updates billing mode for managed DynamoDB lock table to PAY_PER_REQUEST

### PR DESCRIPTION
**Pull request by ThatGerber**

### CODE REVIEW

**Comment by yorinasub17** [comment]
Note to other maintainers: See https://github.com/gruntwork-io/terragrunt/pull/934#discussion_r341253107 for further context and confirmation that `PAY_PER_REQUEST` billing is available in all active regions.

File: dynamodb/dynamo_lock_table.go [hunk 1]
```diff
@@ -78,27 +77,26 @@ func LockTableCheckSSEncryptionIsOn(tableName string, client *dynamodb.DynamoDB)
...
+			{
+				AttributeName: aws.String(ATTR_LOCK_ID),
+				AttributeType: aws.String(dynamodb.ScalarAttributeTypeS),
+			},
+		},
+		KeySchema: []*dynamodb.KeySchemaElement{
+			{
+				AttributeName: aws.String(ATTR_LOCK_ID),
+				KeyType:       aws.String(dynamodb.KeyTypeHash),
+			},
```
Comment (yorinasub17) [inline dynamodb/dynamo_lock_table.go]: Style Nit: Can you revert these changes back to what it was, where we extracted these lists as vars first? It reduces cyclomatic complexity and is easier to read.

Comment (ThatGerber) [reply dynamodb/dynamo_lock_table.go]: Updated.
